catalog/files/impl/src/main/java/org/projectnessie/catalog/files/s3/StsClientsPool.java

```diff
@@ -58,6 +59,7 @@ public StsClientsPool(
     this.clientBuilder = clientBuilder;
     this.clients =
         Caffeine.newBuilder()
+            .scheduler(Scheduler.systemScheduler())
```

Comment:
(ignored as no expiration, harmless)

Reply:
@snazy : shall we revert this change then (for simplicity)?